### DISCUSSION TITLE
Profile website links not interactive

### DISCUSSION THREAD

**Discussion by ykrx** [community/community#55708] [Mobile]
### Select Topic Area

Bug

### Body

Website links on profiles are not tappable / don’t open.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.112.1 (99233639)
Device: iPhone 12 (iOS 16.4.1)
</details>

**Comment by JesusCastroFernandez** [comment]
If the profile website links on a website are not interactive, it could be due to various reasons. Here are some potential causes and solutions to consider:

Incorrect HTML Markup: Check the HTML markup of the website to ensure that the profile website links are correctly coded as clickable links. They should be enclosed within <a> tags with the appropriate href attribute.

CSS Styling: Inspect the CSS styling applied to the profile website links. If there are any conflicting styles or CSS rules that prevent the links from being interactive, you may need to modify the styles to allow for proper interactivity.

JavaScript Interference: If there is JavaScript code running on the website that interferes with the links' functionality, you may need to debug and identify any conflicts or errors in the JavaScript code.

Disabled Links: Double-check if the links have been intentionally disabled or if there are any event handlers attached to them that prevent their interactivity. Look for any JavaScript code that disables or modifies the behavior of the links.

Browser Compatibility: Test the website on different browsers to see if the issue is specific to a particular browser. Ensure that the website and its components are compatible with a wide range of browsers to provide a consistent and interactive user experience.

Content Management System (CMS) Limitations: If the website is built on a CMS platform, there may be limitations or settings within the CMS that affect the interactivity of links. Review the CMS documentation or consult with the website administrator to understand and resolve any limitations.

By investigating these potential causes and implementing the appropriate solutions, you should be able to restore interactivity to the profile website links on your website.

**Reply by ykrx** [reply]
The GitHub mobile app not opening profile links has nothing to do with any of what you (or rather GPT) wrote. Please don’t clog questions up with lazy, irrelevant, generated answers.
